404.html

```diff
@@ -191,7 +191,7 @@
 						<div class="col-sm-8">
 							<h1 class="text-white">404<br>Not Found</h1>
 							<p class="lead text-white">Oh dear, this Page is not Available</p>
-							<a href="index.html" class="btn btn-white btn-hollow">Take Me Home</a>
+							<a href="https://2019.fossasia.org/index.html" class="btn btn-white btn-hollow">Take Me Home</a>
```

Comment:
don't do this

Reply:
Please see https://github.com/fossasia/2019.fossasia.org/pull/55/files#r238092375.